worldengine/simulations/precipitation.py

```diff
@@ -40,20 +45,46 @@ def _calculate(seed, width, height):
         octaves = 6
         freq = 64.0 * octaves
 
+        n_scale = 1024 / float(height) #This is a variable I am adding. It exists
+                                       #so that worlds sharing a common seed but
+                                       #different sizes will have similar patterns
```

Comment:
Common seed and different size? To me it looks more like common size and different seed since n_scale depends on ```height```.

This file, too, would have to be changed a bit after a rebase (lines 41 and 42 will change, I think).